src/pointerMixin.js

```diff
@@ -33,6 +42,20 @@ export default {
       this.pointerAdjust()
     },
     isOpen () {
+      if (!this.hideSelected && this.scrollToSelected && this.value) {
+        let value = JSON.parse(JSON.stringify(this.value))
```

Comment:
Let’s try to avoid removing the reference from the value. Would be best to approach this problem without having to mutate anything. Also, doing parse/stringify will throw errors with recursive objects that might sometimes be a case.

Reply:
A multiselect array will scroll to the first item in the values array. If this is inadequate, do you have any suggestions?
Do you have a suggestion for the isOpen block other than not using parse/stringify?